visidata/features/errors_guide.py

```diff
@@ -0,0 +1,35 @@
+from visidata import GuideSheet, vd
+
+class ErrorsGuide(GuideSheet):
+    guide_text='''# Errors
+
+There are five levels for sidebar status updates: [:status]"debug"[/], [:status]"aside"[/], [:status]"status"[/], [:warning]"warning"[/], and [:error]"error"[/].
+
+[:status]Status[/] is meant to display neutral information.
```

Comment:
I added these, because I mentioned them, but maybe I should just say, "this document focuses on errors + warnings". There is going to be a guide on the Sidebar.